packages/brand-moment/package.json

```diff
@@ -45,6 +45,7 @@
   },
   "peerDependencies": {
     "@kaizen/design-tokens": "^2.8.2",
+    "@kaizen/draft-select": "^1.16.2",
```

Comment:
Correct me if I'm wrong, but this looks like it's only used in stories. Should this instead be a `devDependency`?

Reply:
You're right